src/components/Form/Form.styled.jsx

```diff
@@ -17,6 +17,21 @@ export const FormContainer = styled.form`
 export const ButtonContainer = styled.div`
   display: flex;
   align-items: center;
+  @media (min-width: 600px) {
+    width: 100%;
+  }
+  @media (max-width: 1025px) {
+    width: 70%;
+  }
+  @media (max-width: 500px) {
+    width: 70%;
+  }
+  @media (max-width: 360px) {
+    width: 65%;
+  }
```

Comment:
When you use percentage or relative size units, please add lower and upper bound, as a good practice, for example:

`min-width` and `max-width`

@Katsu08  please check good values to set here